Source/Core/DolphinLibretro/Video.cpp

```diff
@@ -297,9 +310,96 @@ void ContextReset(void)
 
     return;
   }
-#endif
-  if(Video_InitializeBackend())
```

Comment:
Forgot to restore that if statement after that `#ifdef _WIN32` block, making Vulkan crash.